_disability-benefits/claims-process/claim-types/predischarge-claim/tools.md

```diff
@@ -1,29 +1,6 @@
 ---
 title: What tools are there to help me prepare my claim before my discharge?
-template: 6-info-page
+template: 1-topic-landing
 ---
 
-<div class="main" role="main" markdown="0">
-
-<div class="va-action-bar--header">
-  <div class="row">
-    <div class="small-12 columns">
-      <a class="usa-button-primary va-button-primary" href="/disability-benefits/apply-for-benefits/">Apply for Disability Benefits</a>
-    </div>
-  </div>
-</div>
-
-<div class="section one" markdown="0">
-<div class="primary" markdown="0">
-<div class="row" markdown="0">
-<div class="small-12 columns" markdown="1">
-
 Coming soon.
```

Comment:
I think so, yes. I erred on the side of leaving it around in case it's linked to from somewhere. But we can remove the links too.